dts/riscv/rv32m1_ri5cy.dtsi

```diff
@@ -52,23 +52,57 @@
  */
 
 &intmux0 {
+	status = "okay";
+};
+
+&intmux0_ch0 {
+	interrupt-parent = <&event0>;
```

Comment:
Should this be `intmux0` instead of `event0`?

Reply:
Why?  The chaining of interrupts goes `device` -> `intmux0_ch0` -> `event0`.  With this change `intmux0` is just a container device, not an `interrupt-controller` anymore.

Reply:
Ok, thanks for clarifying